.travis.yml

```diff
@@ -4,17 +4,13 @@ go_import_path: go.uber.org/multierr
 
 env:
   global:
-    - GO15VENDOREXPERIMENT=1
     - GO111MODULE=on
 
 go:
-  - 1.11.x
   - 1.12.x
   - 1.13.x
```

Comment:
I would drop unsupported versions; Go release policy states that only the most recent two minor releases are supported.

```suggestion
```